packages/vuelidate/src/core.js

```diff
@@ -94,7 +94,11 @@ function normalizeValidatorResponse (result) {
 function createComputedResult (rule, model, $dirty) {
   return computed(() => {
     if (!$dirty.value) return false
-    const result = callRule(rule, model)
+    let result = callRule(rule, model)
+    if (isPromise(result)) {
+      result = false
+      console.error('Async validators must have the $async property')
```

Comment:
Think this warning will suffice?

Reply:
Might be better to just throw an actual error. That’s what they are for.

Reply:
This will be such a breaking change :(